Public/Src/Engine/Scheduler/PipExecutor.cs

```diff
@@ -2164,14 +2262,107 @@ private static void ReportFileAccesses(ExecutionResult processExecutionResult, F
 
                 RunnableFromCacheResult runnableFromCacheResult;
 
+                bool isCacheHit = cacheHitData != null;
+
+                if (!isCacheHit)
+                {
+                    var pathSetCount = strongFingerprintComputationList.Count;

```

Comment:
>var pathSetCount = strongFingerprintComputationList.Count; [](start = 20, length = 58)

this is the # of (pathSet, SFP) entries, the actual number of unique path sets might be smaller. Use numPathSetsDownloaded instead?